backend/server-connection/src/main/java/org/sonarsource/sonarlint/core/serverconnection/storage/PluginsStorage.java

```diff
@@ -128,4 +128,22 @@ private List<File> getUnknownFiles() {
       return Collections.emptyList();
     }
   }
+
+  public void setReady() {
```

Comment:
I suppose the `setReady` name refers to the analysis readiness? This class is about the storage itself, pretty low-level, so I would avoid a name referring to how it is used. Maybe something like `setEmpty` or `storeNoPlugins` (I'm not good at naming...)

Reply:
Yes, I was thinking of analysis readiness. I will go with the `storeNoPlugins` option then 👍